### PR TITLE
my_account page

### PR DESCRIPTION
**Pull request by quentingigon**
Functionalities are done, it needs a design pass now

### CODE REVIEW

**Review by ecino — APPROVED** [review]

File: crowdfunding_compassion/templates/crowdfunding_components.xml [hunk 1]
```diff
@@ -185,55 +185,10 @@
             </div>
         </template>
 
-         <template id="project_card" name="Project card">
```
Comment (ecino) [inline crowdfunding_compassion/templates/crowdfunding_components.xml:190]: Why did you remove this template? It's used

File: crowdfunding_compassion/templates/crowdfunding_components.xml [hunk 2]
```diff
@@ -185,55 +185,10 @@
...
-                <img class="card-img-top" src="/theme_crowdfunding/static/src/img/musk_ghana_3.jpg" alt="Card image cap" />
-
-                <div class="card-body">
-                    <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
-                    <a href="#" class="btn btn-primary">Go somewhere</a>
-                </div>
-            </div>
-        </template>
-
-             <template id="fund_card" name="Fund card">
```
Comment (ecino) [inline crowdfunding_compassion/templates/crowdfunding_components.xml:224]: Why did you remove this template?

**Review by ecino — CHANGES_REQUESTED** [review]

File: crowdfunding_compassion/templates/crowdfunding_components.xml (hunk 1 above)
Comment (quentingigon) [reply crowdfunding_compassion/templates/crowdfunding_components.xml:190]: It was not removed but moved to the templates folder

File: crowdfunding_compassion/templates/crowdfunding_components.xml (hunk 2 above)
Comment (quentingigon) [reply crowdfunding_compassion/templates/crowdfunding_components.xml:224]: It is a mistake, I will put it back

File: crowdfunding_compassion/templates/crowdfunding_components.xml (hunk 1 above)
Comment (ecino) [reply crowdfunding_compassion/templates/crowdfunding_components.xml:190]: You probably forgot to commit the new file because here in the diff I only see the supression. Please check again

Comment (quentingigon) [reply crowdfunding_compassion/templates/crowdfunding_components.xml:190]: Sorry I forgot that I renamed it to crowdfunding_project_card, in crowdfunding_components.xml

Comment (quentingigon) [reply crowdfunding_compassion/templates/crowdfunding_components.xml:190]: in homepage there was a call to project_card, so I changed it

Comment (ecino) [reply crowdfunding_compassion/templates/crowdfunding_components.xml:190]: Ok, good :+1:

File: crowdfunding_compassion/templates/myaccount_crowdfunding_page.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+<odoo>
+    <data>
+        <template id="myaccount_crowdfunding_view_template" name="MyAccount crowdfunding template">
+            <t t-call="website.layout">
```
Comment (ecino) [inline crowdfunding_compassion/templates/myaccount_crowdfunding_page.xml]: Should be changed because of my corrections of the templates.
```suggestion
            <t t-call="crowdfunding_compassion.layout">
```

File: crowdfunding_compassion/templates/myaccount_crowdfunding_page.xml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        </template>
+
+        <record id="myaccount_crowdfunding_view" model="ir.ui.view">
+            <field name="name">MyAccount view</field>
+            <field name="type">qweb</field>
+            <field name="key">crowdfunding.myaccount</field>
+            <field name="arch" type="xml">
+                <t t-call="crowdfunding_compassion.myaccount_crowdfunding_view_template"/>
+            </field>
+        </record>
```
Comment (ecino) [inline crowdfunding_compassion/templates/myaccount_crowdfunding_page.xml]: This is useless. The `<template>` tag is a shorthand for creating the `ir.ui.view` record.
```suggestion
```

File: crowdfunding_compassion/templates/myaccount_crowdfunding_page.xml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        <record id="myaccount_crowdfunding_page" model="website.page">
+            <field name="url">/my_account</field>
+            <field name="website_indexed" eval="True"/>
+            <field name="website_id" eval="ref('crowdfunding_compassion.crowdfunding_website')"/>
+            <field name="website_published">True</field>
+            <field name="name">MyAccount</field>
+            <field name="type">qweb</field>
+        </record>
+
+        <template id="my_account_crowdfunding_link" name="Link to MyAccount page" inherit_id="portal.frontend_layout">
```
Comment (ecino) [inline crowdfunding_compassion/templates/myaccount_crowdfunding_page.xml]: I think this mean you will change the regular my account page as well. Don't think it's the right way to do it. I think we have a separate navbar for the crowdfunding website and we should have a user icon here with the link to the /my_account, and not touch the default my home menu for the general website.

File: crowdfunding_compassion/templates/myaccount_crowdfunding_participant_update_page.xml [hunk 6]
```diff
@@ -0,0 +1,32 @@
+<odoo>
+    <data>
+        <template id="participant_update_view_template">
+            <t t-call="website.layout">
```
Comment (ecino) [inline crowdfunding_compassion/templates/myaccount_crowdfunding_participant_update_page.xml]:
```suggestion
            <t t-call="crowdfunding_compassion.layout">
```

File: crowdfunding_compassion/templates/myaccount_crowdfunding_participant_update_page.xml [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+                <div t-attf-class="{{form.form_wrapper_css_klass}} center">
+                    <t t-raw="form.form_render()"/>
+                </div>
+
+                <t t-call="crowdfunding_compassion.crowdfunding_footer"/>
```
Comment (ecino) [inline crowdfunding_compassion/templates/myaccount_crowdfunding_participant_update_page.xml]:
```suggestion
```
No more needed after my changes for templates.

File: crowdfunding_compassion/templates/myaccount_crowdfunding_participant_update_page.xml [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+        </template>
+
+        <record id="participant_update_view" model="ir.ui.view">
+            <field name="name">Participant update view</field>
+            <field name="type">qweb</field>
+            <field name="key">crowdfunding.participant_update</field>
+            <field name="arch" type="xml">
+                <t t-call="crowdfunding_compassion.project_participant_view_template"/>
+            </field>
+        </record>
```
Comment (ecino) [inline crowdfunding_compassion/templates/myaccount_crowdfunding_participant_update_page.xml]:
```suggestion
```

File: crowdfunding_compassion/templates/myaccount_crowdfunding_project_update_page.xml [hunk 9]
```diff
@@ -0,0 +1,32 @@
+<odoo>
+    <data>
+        <template id="project_update_view_template">
```
Comment (ecino) [inline crowdfunding_compassion/templates/myaccount_crowdfunding_project_update_page.xml]: This is exactly the same template as `participant_update_view`. You can rename it to more generic name like `crowdfunding_form_template` and use the same template for the two pages. The only difference will be in the controller where you will inject another form in the template.

**Review by ecino — CHANGES_REQUESTED** [review]

File: crowdfunding_compassion/templates/myaccount_crowdfunding_page.xml (hunk 5 above)
Comment (quentingigon) [reply crowdfunding_compassion/templates/myaccount_crowdfunding_page.xml]: Yes I did that when I was learning the system and it's not useful anymore so I will delete it
